app/layout.tsx

```diff
@@ -8,8 +8,9 @@ const baloo = Baloo_2({
 });
 
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "PoiPoi",
+  description:
+    " immersive web game that challenges players to identify locations within Tokyo using visual clues from carefully curated images.",
```

Comment:
Can we add "`An` immersive web game that challenges players to identify locations `on an interactive map` ~within Tokyo~ using visual cues from carefully curated images and find those location in the real world."